src/main/resources/citizen/templates/document/defendantResponseReceipt.html

```diff
@@ -457,7 +457,7 @@
   </div>
 </div>
 
-{% if paymentOption == 'By instalments' or paymentOption == 'By a set date' %}
+{% if (paymentOption == 'By instalments' or paymentOption == 'By a set date') and  statementOfMeans is defined %}
```

Comment:
you need to populate `statementOfMeans`, otherwise you have broken the individual flow.